flutter/news/test/widgets/article_card_test.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:network_image_mock/network_image_mock.dart';
+import 'package:news/features/frontpage/presentation/bloc/top_headlines_viewstate.dart';
+import 'package:news/features/frontpage/presentation/widgets/news_card.dart';
+
+import '../core/utils/extensions.dart';
+
+void main() {
+  final article = TopHeadlineViewState.from(
+    article: Stub.article(
+      title: "1",
+    ),
+  );
+
+  testWidgets(
```

Comment:
any link or good resource that you know for learning more about widget testing?

Reply:
i guided myself by here https://docs.flutter.dev/cookbook/testing/widget/introduction